plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -191,7 +191,21 @@ global proc int mayaUsdTranslatorExport (string $parent,
             checkBoxGrp -l "Color Sets:" -annotation $colorSetsAnn exportColorSetsCheckBox;
 
             checkBoxGrp -l "UV Sets:" -annotation $uvSetsAnn exportUVsCheckBox;
-            
+
+            string $exporters[] = `mayaUSDListShadingModes -export`;
+            optionMenuGrp -l "Materials:" shadingModePopup;
+            for ($modeName in $exporters) {
+                string $niceName = $modeName;
+                if ($modeName == "displayColor") {
+                    $niceName = "Material Colors";
+                } else if ($modeName == "pxrRis") {
+                    $niceName = "RfM Shaders";
+                } else if ($modeName == "useRegistry") {
+                    $niceName = "Preview Surface Shaders";

```

Comment:
Again, this is ok as a placeholder for now, but we might want some symmetry between import and export. Looks like import uses "USD Shade"?